dev-python/python-caja/python-caja-1.22.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MATE_LA_PUNT="yes"
+PYTHON_COMPAT=( python2_7 python3_6 python3_7 )
```

Comment:
Please drop python2_7 support

Reply:
As several extensions don't support py3 yet, I'm inclined not to drop it yet.

Reply:
Also, upstream didn't remove it. The next version 1.23 removed python 2.
@NP-Hardass I have prepared updated ebuilds in my overlay. Can you merge this so I can add more PRs with the next ebuilds?